hera_cal/lst_stack/config.py

```diff
@@ -665,6 +668,84 @@ def _read(cls, group: h5py.Group):
         return cls(**dct)
 
 
+class LSTBinConfiguratorSingleBaseline():
+    '''This is a much simpler, less fully featured version of LSTBinConfigurator,
+    designed for LST-binning single baseline (generally 4-pol) files together
+    across nights. Its main goal is '''
```

Comment:
Let's fix up (and finish) the docstring.

Reply:
Also, let's document that if we have redundantly averaged baselines, this class requires that the keyed baseline on each night needs to be the same.

Reply:
Oops, my bad. Added those details.